data/updates/6.0/mysql.sql

```diff
@@ -1,10 +0,0 @@
---
--- The `ezrole` table has changed. `version` has been added to the primary key, after the `id`. This is because role
--- drafts are now handled differently. A draft now has the same `id` as the published role, and `version` decides which
--- is which. You should make sure you don't have any role drafts in the database when upgrading. To find drafts, select
--- entries in the `ezrole` table where `version` is not zero. Before this change, the `version` of a draft equals the
--- `id` of the role. The safe way to get rid of stray role drafts is to edit the role in question, and then cancel
--- editing.
---
-
-ALTER TABLE ezrole DROP PRIMARY KEY, ADD PRIMARY KEY(id,version);

```

Comment:
So the answer is; move it to `6.0/unstable/dbupdate-5.4.0-to-6.0.0alpha7.sql` and create one for reverting it for 15.07 users in `6.0/unstable/dbupdate-6.0.0alpha7-to-6.0.0beta1.sql` ;)